frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -759,7 +759,8 @@ def process(self, send_now=False) -> EmailQueue | None:
 		if not queue_separately:
 			recipients = list(set(final_recipients + self.final_cc() + self.bcc))
 			q = EmailQueue.new({**queue_data, **{"recipients": recipients}}, ignore_permissions=True)
-			send_now and q.send()
+			if send_now:
+				frappe.enqueue(q.send, queue="short", enqueue_after_commit=True, at_front_when_starved=True)
```

Comment:
Maybe this should be handled higher up somewhere :eyes: but conceptually this is right thing to do IMO.